lib/model/query/entities.js

```diff
@@ -278,13 +278,19 @@ const _processSubmissionEvent = (event, parentEvent) => async ({ Datasets, Entit
   const dataset = (await Datasets.get(form.get().projectId, entityData.system.dataset, true))
     .orThrow(Problem.user.datasetNotFound({ datasetName: entityData.system.dataset }));
 
-  // Create entity
-  if (entityData.system.create === '1' || entityData.system.create === 'true')
-    return Entities._createEntity(dataset, entityData, submissionId, submissionDef, submissionDefId, event, parentEvent);
-
   // Or update entity
```

Comment:
Update or remove comment?

Reply:
Specifically, I think the "Or" seems out-of-place now that "Create entity" above is gone.